src/DBmysql.php

```diff
@@ -1622,12 +1622,15 @@ public function deleteOrDie($table, $where, $message = '', array $joins = [])
      *
      * @since 10.0.0
      *
-     * @param string $table  Table name
+     * @param string $table Table name
      *
      * @return mysqli_result|boolean Query result handler
+     *
+     * @deprecated 10.0.9
      */
     public function truncate($table)
     {
+        Toolbox::deprecated();
```

Comment:
We should probably wait for GLPI 10.1 to mark it as deprecated. This method is used, at least in `formcreator`, and many people would probably complains about "error" during its install/update.

Reply:
ok...